pkg/operator/controller/controller_lb.go

```diff
@@ -97,44 +92,24 @@ func (r *reconciler) currentLoadBalancerService(ci *operatorv1.IngressController
 	return service, nil
 }
 
-// finalizeLoadBalancerService deletes any DNS entries associated with any
-// current LB service associated with the ingresscontroller and then finalizes the
-// service.
-func (r *reconciler) finalizeLoadBalancerService(ci *operatorv1.IngressController, dnsConfig *configv1.DNS) error {
+// finalizeLoadBalancerService removes finalizers from any LB service. This used
+// to be to help with DNS cleanup, but now that's no longer necessary, and so we
+// just need to clear the finalizer which might exist on existing resources.
+//
+// TODO: How can we delete this code?
```

Comment:
Call `finalizeLoadBalancerService` on every ingresscontroller when the operator first starts, then delete that logic after a couple releases?

Reply:
Even before that, we can delete `enforceIngressFinalizer` now, right?

Reply:
I don't think so? Do you mean removing the finalizer from new LB services we create? The entire cleanup strategy revolves around ingresscontroller finalizers.

Reply:
Sorry, I mixed up my finalizers.  I mean remove the finalizer from any existing LB service on start-up, and stop adding one in `desiredLoadBalancerService`.